app/services/events/validate_creation_service.rb

```diff
@@ -1,56 +1,26 @@
 # frozen_string_literal: true
 
-# frozen_string_literal: true
-
 module Events
   class ValidateCreationService
```

Comment:
Why do we still need this class? 🤔

Reply:
I think it is still used for the "fees#estimate"

Reply:
yes it's still used by the `fees#estimate` endpoint